datadog/1.11/README.md

```diff
@@ -0,0 +1,207 @@
+
+For certain services, such as Redis, Elasticsearch, and Apache server (httpd), the Datadog Agent can connect without any additional setup in most cases. (See the [Autodiscovery docs][autodiscovery] for the full list.) If you're running one of those services, you should start seeing metrics in Datadog within minutes (on [the Redis dashboard][redis-dash], for instance).
+
+### Monitoring services with custom configurations
+
+For other services, you can create monitoring configuration templates that Datadog will apply whenever it detects the service running on a node in your cluster. For production use, you will likely want to [use a key-value store][autodiscovery] such as etcd or Consul to manage these templates for you. To demonstrate the basics of service discovery, though, here we'll walk through a simple example using configuration templates stored in a directory on the host that is mounted by the Datadog Agent container.
+
+#### Create a config template
+
+On your agent nodes, create a directory to house your custom configs and create a YAML file for the service you want to monitor (drawing on [the example YAML templates][yamls] that ship with the Datadog Agent). In this example we'll set up service discovery for MYSQL, which requires a password to access database metrics:
+
+```bash
+# get a node ID
+dcos node
+# connect to a node selected from the previous command's output
+dcos node ssh --master-proxy --mesos-id=225ca9ef-f1bc-43c1-bf31-b9330f89bb50-S0
+# create a config folder for the mysql integration
+mkdir -p /opt/datadog-agent-conf.d/mysql.d/
+# create a config file for AutoDiscovery in this folder
+touch /opt/datadog-agent-conf.d/mysql.d/auto_conf.yaml
+```
+
+Open your newly created `auto_conf.yaml` file and paste in [the basic config][mysql-docs] template from the Datadog Agent, then add a `ad_identifiers` section at the top to tell Datadog which Docker images this template applies to (for this example we'll assume that your MySQL database already has [a `datadog` user created][mysql-docs] with the necessary permissions):
+
+```yaml
+ad_identifiers:
+  - mysql
+
+init_config:
+
+instances:
+  - server: "%%host%%"
+    user: datadog
+    pass: password_for_datadog_user_in_db
+
+```
+
+#### Apply the config template
+
+In order to apply your newly created config template, you need to mount the config files on the DC/OS agent node into the Datadog container running on that host.
+
+To pass custom volumes to the Datadog service you will need to create the service from scratch as Universe packages don't allow changing mounted volumes. In this new service, under "Volumes" add the following volume on top of the default ones (as shown on the image below):
```

Comment:
I ran through this today, and I'm concerned there isn't enough detail here to have someone replicate what you're doing to create this custom service. A new service doesn't have any default volumes, and you'll be required to pass along all the other details as well.

I hate to suggest this, since the Marathon UI isn't well-supported, but I think we should adjust these instructions to suggest using the marathon UI (at /marathon in the DC/OS UI) to edit this, as in this screenshot.
![screenshot_2018-07-03_07-33-43](https://user-images.githubusercontent.com/879924/42226326-8c7c668a-7e93-11e8-8759-59cbaaf4f718.png)